lib/mayaUsd/render/MaterialXGenOgsXml/libraries/mx_lighting_maya_v3.glsl

```diff
@@ -22,8 +22,8 @@ vec3 mx_environment_radiance(vec3 N, vec3 V, vec3 X, vec2 alpha, int distributio
     }
     
     // Generate tangent frame.
-    vec3 Y = normalize(cross(N, X));
-    X = cross(Y, N);
+    X = normalize(X - dot(X, N) * N);
+    vec3 Y = cross(N, X);
```

Comment:
Took the latest [mx_environment_fis.glsl](https://github.com/AcademySoftwareFoundation/MaterialX/blob/main/libraries/pbrlib/genglsl/lib/mx_environment_fis.glsl) from MaterialX and adapted it for Maya lighting.